sdk/remoterendering/azure-mixedreality-remoterendering/azure/mixedreality/remoterendering/_remote_rendering_client.py

```diff
@@ -52,15 +54,25 @@ class RemoteRenderingClient(object):
     :param credential: Authentication for the Azure Remote
         Rendering account. Can be of the form of an AzureKeyCredential, TokenCredential or an AccessToken acquired
         from the Mixed Reality Secure Token Service (STS).
-    :type credential: Union[TokenCredential, AzureKeyCredential, AccessToken]
+    :type credential: Union[AzureKeyCredential, TokenCredential, AccessToken]
     :keyword api_version:
         The API version of the service to use for requests. It defaults to the latest service version.
         Setting to an older version may result in reduced feature compatibility.
-    :paramtype api_version: str or ~azure.mixedreality.remoterenderings.RemoteRenderingApiVersion
+    :type api_version: str or ~azure.mixedreality.remoterenderings.RemoteRenderingApiVersion
+    :keyword polling_interval:
+        Seconds to wait between each check, whether the session is ready yet.
+    :keyword authentication_endpoint_url:
+        Overwrite for the authentication endpoint. Usually using account_domain for the authentication domain is enough.
+        If used, specify the whole authentication url including the schema.
     """
 
-    def __init__(self, endpoint, account_id, account_domain, credential, **kwargs):
-        # type: (str, str, str, Union[TokenCredential, AccessToken], Any) -> None
+    def __init__(self,
+                 endpoint: str,
+                 account_id: str,
+                 account_domain: str,
+                 credential: Union[AzureKeyCredential, 'TokenCredential', AccessToken],
+                 **kwargs) -> None:
+
         self._api_version = kwargs.pop(
             "api_version", RemoteRenderingApiVersion.V2021_01_01
```

Comment:
You can use something like https://github.com/Azure/azure-sdk-for-python/blob/main/sdk/search/azure-search-documents/azure/search/documents/_api_versions.py#L17 then if you have multiple api versions, you don't need to update them one by one.